pages/TheExplorationGallery.html

```diff
@@ -11,32 +11,39 @@
   <p>
```

Comment:
General comment: throughout this page, Gallery should be changed to Library (not only in the text, but also in the filename, page title, and the sidebar link). Other pages should be fixed as well, if necessary.